slack-dark-mode.sh

```diff
@@ -22,7 +22,7 @@ if [[ -z $OSX_SLACK_RESOURCES_DIR ]] || [[ -z $LINUX_SLACK_RESOURCES_DIR ]]; the
 
 echo && echo "This script requires sudo privileges." && echo "You'll need to provide your password."
 
-type npx
+NPX_PATH=$(type -P npx)
 if [[ "$?" != "0" ]]; then echo "Please install NodeJS for your OS." && echo "macOS users will also need to install Homebrew from https://brew.sh" && exit 1; fi
```

Comment:
That's going to have to change, since after your change it won't ever get to this line.